tests/link/pytorch/test_blockwise.py

```diff
@@ -12,7 +12,7 @@
 basic = pytest.importorskip("pytensor.link.pytorch.dispatch.basic")
 
 
-class TestOp(Op):
+class BatchedTestOp(Op):
```

Comment:
We can reverse this, i was just getting annoyed with the pytest warning

Reply:
Makes sense, just make sure to be in a separate commit